dev/docker/Makefile

```diff
@@ -23,20 +23,27 @@ build: $(CCACHE_DIR) $(VCPKG_BINARY_CACHE_DIR) $(MAVEN_M2_DIR)
 
 docker-image:
 	docker build \
-		--file docker/Dockerfile \
+		--file Dockerfile \
 		--build-arg BUILDER_UID=`id -u` \
 		--build-arg BUILDER_GID=`id -g` \
 		--tag "$(DOCKER_IMAGE)" \
 		.
 
-docker-image-gha:
+docker-image-static:
 	docker build \
-		--file docker/Dockerfile.gha \
+		--file Dockerfile.centos7-static-build \
 		--tag "$(DOCKER_IMAGE)" \
 		--build-arg HTTPS_PROXY="" \
 		--build-arg HTTP_PROXY="" \
 		.
 
+docker-image-dynamic:
+	docker build \
+		--file Dockerfile.centos8-dynamic-build \
```

Comment:
Docker file is `Dockerfile.centos8-dynamic-build` while variable `DOCKER_IMAGE` is `apache/gluten:vcpkg-centos-7`. Something wrong here?